src/orchard/complete.clj

```diff
@@ -0,0 +1,25 @@
+(ns orchard.complete
+  (:require [cljs-tooling.complete :as cljs-complete]
+            [compliment.core :as jvm-complete]
+            [compliment.utils :as jvm-complete-utils]
+            [orchard.misc :as u]
+            [orchard.util.cljs :as cljs]))
+
+(defn complete
```

Comment:
The API of those functions has to be changed to be non-nREPLy. For normally callers something like a `msg` doesn't make much sense here and probably this can just have normal positional arguments.

Reply:
Sure, but I wasn't sure how to remove the `msg` due to the `(cljs/grab-cljs-env msg)`.